contributors/chairs-and-techleads/process-workflow-guide.md

```diff
@@ -0,0 +1,253 @@
+# Process and Workflow Guide
+
+Author: **[@LappleApple](https://github.com/lappleapple)** || Contributors: **[@annajung](https://github.com/annajung)**, **[@kikisdeliveryservice](https://github.com/kikisdeliveryservice)**, **[@neolit123](https://github.com/neolit123)** 
+
+## Who This Is For
+
+- SIG leads who want guidance on how to structure workflow and processes using models explored successfully by other SIGs 
```

Comment:
```suggestion
- Leads who want guidance on how to structure workflow and processes using models explored successfully by other groups 
```
Might be nitty, but SIGs are our main group, but this doc would be applicable for WGs and subproject. Language would need to be updated throughout the doc though.